packages/app-builder-lib/src/publish/BitbucketPublisher.ts

```diff
@@ -18,11 +18,13 @@ export class BitbucketPublisher extends HttpPublisher {
     super(context)
 
     const token = process.env.BITBUCKET_TOKEN
+    const username = process.env.BITBUCKET_USERNAME
+
     if (isEmptyOrSpaces(token)) {
       throw new InvalidConfigurationError(`Bitbucket token is not set using env "BITBUCKET_TOKEN" (see https://www.electron.build/configuration/publish#BitbucketOptions)`)
     }
     this.info = info
-    this.auth = BitbucketPublisher.convertAppPassword(this.info.owner, token)
+    this.auth = BitbucketPublisher.convertAppPassword(this.info.owner, token, username)
```

Comment:
I would recommend keeping the same function signature since it's already explicit and instead do:
```
this.auth = BitbucketPublisher.convertAppPassword(username ?? this.info.owner, token)
```